c/include/ml-api-service.h

```diff
@@ -190,6 +190,37 @@ int ml_service_destroy (ml_service_h handle);
  */
 int ml_service_get_pipeline_state (ml_service_h handle, ml_pipeline_state_e *state);
 
+/**
+ * @brief Creates query client service handle with given ml-option handle.
+ * @since_tizen 7.0
+ * @remarks The @a handle should be destroyed using ml_service_destroy().
+ * @param[in] option The option handle used for creating query client.
+ * @param[out] handle Newly created service handle is returned.
+ * @return @c 0 on Success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Fail. The parameter is invalid.
+ * @retval #ML_ERROR_OUT_OF_MEMORY Failed to allocate required memory.
+ * @retval #ML_ERROR_STREAMS_PIPE Failed to launch the pipeline.
+ */
+int ml_service_query_create_client (ml_option_h option, ml_service_h *handle);
```

Comment:
Discussion: ml_service_query_create() vs ml_service_query_create_client()
(e.g. service_query_create() > service_query_request() > service_destroy())

Reply:
`ml_service_query_create()` seems good to me.